scalafix-tests/integration/src/test/scala/scalafix/tests/interfaces/ScalafixArgumentsSuite.scala

```diff
@@ -165,8 +168,8 @@ class ScalafixArgumentsSuite extends AnyFunSuite with DiffAssertions {
           |
           |object Semicolon {
           |  val a = 1; // みりん þæö
-          |  implicit val b = List(1)
-          |  def main { println(42) }
```

Comment:
this is not valid syntax in Scala 3, so switching from ProcesureSyntax to RedundantSyntax